tfx/components/transform/executor_test.py

```diff
@@ -41,10 +43,39 @@ class _TempPath(types.Artifact):
 # TODO(b/122478841): Add more detailed tests.
 class ExecutorTest(tft_unit.TransformTestCase):
 
-  def _get_source_data_dir(self):
-    return os.path.join(
+  @classmethod
```

Comment:
I did some updates to this file, could you sync to head?

Reply:
synced. now using `_get_dataset_size()` method.